### DISCUSSION TITLE
image quality problem

### DISCUSSION THREAD

**Discussion by melbrtd** [arkypita/LaserGRBL#1879] [Q&A]
Hello, I need your precious help for a problem that I have had for a few days.
My laser doesn't burn as well as it used to when it's only 2 months old. The engraving depth is different and the quality has deteriorated. I thought it was the laser but I realize that when I upload my illustration to GRBL, it is already transmitted in poor quality. The red lines are interspersed with white lines while I have never changed the properties. 

Also, I had illustrations in 13x13cm that I made on Canva and which passed very well on GRBL and now the dimension is no longer accepted because supposedly too large compared to the laser, while not at all..

In photo 1, my engraving was perfect a few weeks ago. Now, as in the photo 2 can see the engraving much shallower with white lines. I made a screenshot to show you how the illustration looks on GRBL and it's really bad. Before the red lines were very full and sharp.
I changed the properties, power and speed but nothing changes.

 Thank you in advance for your help and sorry for my vocubular, I am French :) 

![Photo 2](https://user-images.githubusercontent.com/110282297/181903368-e1dfb35f-fefa-4401-bc68-0f65939f85b4.jpg)
![Screenshot](https://user-images.githubusercontent.com/110282297/181903522-b3261007-42a6-49c2-84b9-c9a40c618f2f.png)

![Photo 1](https://user-images.githubusercontent.com/110282297/181903318-c7808cc9-2005-4578-8c6a-f9157e793625.jpg)

**Comment by Tribo-2dot0** [comment]
with how many lines/mm are you defining?
